metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_profiling.py

```diff
@@ -0,0 +1,189 @@
+import logging
+import re
+from typing import Any, Dict, List, Tuple
+
+from datahub.ingestion.source.dremio.dremio_api import DremioAPIOperations
+from datahub.ingestion.source.dremio.dremio_config import ProfileConfig
+
+logger = logging.getLogger(__name__)
+
+
+class DremioProfiler:
```

Comment:
Would be good to integrate `DatahubGEProfiler` here instead to make use of other improvements(multithreading, query optimisation, etc) around it. Not sure if this needs to be done right away. cc: @hsheth2 to weigh in.

Reply:
Decided to stick with current native dremio profiling with full support from basic profiling config support.